node/src/actors/chain_manager/actor.rs

```diff
@@ -308,4 +317,19 @@ impl ChainManager {
             })
             .wait(ctx);
     }
+
+    /// Put some basic information into the scope of the telemetry service (Sentry)
+    #[cfg(feature = "telemetry")]
+    fn configure_telemetry_scope(&mut self) {
```

Comment:
What happens if the node panics before calling this function?

Reply:
No info about node identity, but we get the report anyway.

Reply:
Perfect